### ISSUE TITLE
Install fails on go 1.10

### ISSUE THREAD

**Issue by jmayalag** [dedis/prifi#179]
`prifi.sh install` fails when go version is 1.10

**Comment by jmayalag** [comment]
The second capturing group is missing a *
```bash
GO_VER=$(go version 2>&1 | sed 's/.*version go\([[:digit:]]*\)\.\([[:digit:]]\)\(.*\)/\1\2/; 1q')
```
This should do it (while major version <= 9)
```bash
GO_VER=$(go version 2>&1 | sed 's/.*version go\([[:digit:]]*\)\.\([[:digit:]]*\)\(.*\)/\1\2/; 1q')
```

**Comment by lbarman** [comment]
Indeed ! Thanks for your feedback. Fixed aa7fb712f82fc86a24c80188374d8d86d42525a0
